client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-assembler-preview.tsx

```diff
@@ -16,6 +16,16 @@ import type { Pattern } from './types';
 import type { Design } from '@automattic/design-picker';
 import './pattern-assembler-preview.scss';
 
+// Footer alignment
+const inlineCss = `
+	.site-footer-container {
+		margin-block-start: auto;
```

Comment:
Looks like some of CSS are injected twice so that its priority becomes higher than the original style `margin-top: auto`.

![image](https://user-images.githubusercontent.com/13596067/190585374-a9c89b7e-f0d9-48a2-ade8-6449816ed8e8.png)

It would be better if we could solve it. Otherwise, some styles might be overridden in the future and we might need to add more and more inline styles like this.

But we could go with this way first 🙂

Reply:
Nice find 🙌
Let's follow up with that work in another issue. I wouldn't know now where to start investigating that duplication.

Reply:
We'll find that out!